sphinx_gherkindoc/writer.py

```diff
@@ -303,11 +299,15 @@ def steps(steps: List[behave.model.Step], step_format: str = "{}") -> None:
                 text(step.text)
 
     def examples(
-        scenario: behave.model.Scenario, feature: behave.model.Feature
+        example_source: Union[behave.model.Scenario, behave.model.Feature],
+        example_source_parent: Optional[behave.model.Feature] = None,
```

Comment:
I was expecting this change to cause the expected rst in the tests to now actually include the examples table, but I don't see that in the diff. Is there an issue with the tests?